packages/wallets/src/features/cfd/screens/Jurisdiction/JurisdictionScreen.scss

```diff
@@ -52,6 +53,7 @@
         font-size: 14px;
         font-weight: 700;
         gap: 12px;
+        height: 8%;
         justify-content: center;
```

Comment:
These stylings fixed issues with the card overlapping with header:
## Before:
https://github.com/binary-com/deriv-app/assets/103016120/e2ec56b9-0618-4734-8066-03d602cdd6b8

## After:
https://github.com/binary-com/deriv-app/assets/103016120/f7f858d4-5e4d-4e33-845d-f13e497db5f4